src/Card/Card.Content.js

```diff
@@ -0,0 +1,46 @@
+import { dcnb } from "cnbuilder";
+import React from "react";
+import PropTypes from "prop-types";
+import { FlexBox } from "../FlexBox/FlexBox";
+
+export const Content = ({ children, className, align, ...props }) => {
+  // Content alignment including image and CTA, default is left-align
+  // This setting overrides the alignment option in the nested CTA
+  let bodyAlign = "su-items-start";
+  let contentClass = "";
+
+  if (align === "center") {
+    contentClass = dcnb(contentClass, "children:su-mx-auto su-text-center");
+    bodyAlign = "su-items-center";
+  }
```

Comment:
@rebeccahongsf  I just realized 2 things. The plugin that enables the "children" prefix isn't installed in Decanter 7 core. I'm going to open a PR to add that. Also, what do you think about moving the align props to the main "Card" component, since there might be cases that we need to include the alignment of the image (if it's a small image that doesn't span the width of the whole card) 😄

Reply:
Sounds good! I didn't think of the use case of a small card that doesn't span the width of the whole card. 🤔  We can definitely bump the alignment props up. My question though would be did we want the image and body content to inherit the alignment from the main Card component or did we want to have each subcomponent aligned separately (i.e. the developer can then chose to align the image right but have the body centered/left aligned)?

Reply:
@rebeccahongsf Don't worry about this for now 😄 ! I found that this is a lot easier if we have the "children" prefix plugin like we do in the SAA repo, and you got a good point about perhaps people might want the image and content to align differently. Let's table this for now. Your current way stays true to the spirit of the separate subcomponents 😸